src/actions/plugin.js

```diff
@@ -0,0 +1,18 @@
+'use strict'
+
+const { prompt } = require('../dialog')
+
+var plugins
+if (process.type !== 'browser') {
```

Comment:
Yeah, actions being required in main is a part that is not completely thought-through yet. Technically, this means we should move actions to `common` but that didn't feel right. Most actions are simple action creators, but there are a handful of async actions which only work in renderer (and actions, in general, belong to React and should stay in renderer). Something to think about.

Anyway, in this case here it might be preferably to pass the `plugins` in the action's payload (that way the action does not depend on the singleton). While we're at it, it would make sense for the action to have a boolean `prompt` in `meta` so that you can control whether or not to show the prompt.